### PR TITLE
README, releases: add TiDB 2.1.3 releases notes

### PR DESCRIPTION
**Pull request by CaitinChen**
@lilin90 PTAL

### CODE REVIEW

File: releases/2.1.3.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+- Support obtaining the monitoring information using the HTTP method [#3855](https://github.com/tikv/tikv/pull/3855)
+- Fix the NULL issue of `data_format` [#4075](https://github.com/tikv/tikv/pull/4075)
+- Add verifying the range for scan requests [#4124](https://github.com/tikv/tikv/pull/4124)
+
+## Tools
+
++ TiDB-Binlog
+    - Fix the `no available pump` issue while TiDB is started or restarted [#157](https://github.com/pingcap/tidb-tools/pull/158)
+    - Enable outputting the Pump client log [#165](https://github.com/pingcap/tidb-tools/pull/165)
+    - Fix the data inconsistency issue caused by the unique key containing the NULL value when the table only has the unique key and does not have the primary key [#446](https://github.com/pingcap/tidb-binlog/pull/446)
```
Comment (lilin90) [inline releases/2.1.3.md]:
```suggestion
    - Fix the data inconsistency issue caused by the unique key containing the NULL value when the table only has the unique key and does not have the primary key
```

**Comment by CaitinChen** [comment]
@lilin90 PTAL again

File: releases/2.1.3.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    - Fix the panic issue of Generated Column during the process of `UPDATE` in some cases [#8980](https://github.com/pingcap/tidb/pull/8980)
+    - Fix the upper bound overflow issue of the statistics histogram in some cases [#8989](https://github.com/pingcap/tidb/pull/8989)
+    - Support Range for `_tidb_rowid` construction queries, to avoid full table scan and reduce cluster stress [#9059](https://github.com/pingcap/tidb/pull/9059)
+    - Return an error when the `CAST(AS TIME)` precision is too big [#9058](https://github.com/pingcap/tidb/pull/9058)
+    - Allow using `Sort Merge Join` in the Cartesian product [#9037](https://github.com/pingcap/tidb/pull/9037)
+    - Fix the issue that the statistics worker cannot resume after the panic in some cases [#9085](https://github.com/pingcap/tidb/pull/9085)
+    - Fix the issue that `Sort Merge Join` returns the wrong result in some cases [#9046](https://github.com/pingcap/tidb/pull/9046)
+    - Support returning the JSON type in the `CASE` clause [#8355](https://github.com/pingcap/tidb/pull/8355
```
Comment (lilin90) [inline releases/2.1.3.md]:
```suggestion
    - Support returning the JSON type in the `CASE` clause [#8355](https://github.com/pingcap/tidb/pull/8355) 
```

**Comment by CaitinChen** [comment]
@lilin90 PTAL again

**Pull request merged**